vendor.conf

```diff
@@ -33,7 +33,7 @@ github.com/smartystreets/go-aws-auth          8ef1316913ee4f44bc48c2456e44a5c1c6
 github.com/mcuadros/go-version                6d5863ca60fa6fe914b5fd43ed8533d7567c5b0b
 
 github.com/rancher/rdns-server                f79428ee317c10fa75f6bf2846aa6b88bff081ef
-github.com/rancher/types                      374bbecefee300751c27b43a89e5ed988650bf27
+github.com/rancher/types                      af51210fd267134d165a85c58f07a0094c16d19a file:///Users/alena/go/src/github.com/rancher/types/.git
```

Comment:
^^

Reply:
it will get fixed once corresponding types pr makes it in